status-im/status-react
# feature #3443 - automatically loading all dev contacts in nightlies

fixes #3443

### Summary:

Automatically adds all Status devs as contacts in any non-release version. Useful for dogfooding.

### Steps to test:

- Start Status and create an account
- Try sending a message to check if you have any default contacts other than Jarrad. There should be all of us.

- Recover an existing account
- Also see if the contacts are there.

status: ready
